<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Homepage</class>
 <widget class="QDialog" name="Homepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>828</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>50</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255)</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButton_men">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Men</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3_EcoCDeals">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Eco Christmas Deals</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_pic1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>201</width>
     <height>231</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>137</width>
     <height>111</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Downloads/451ac9ac6fd294b8b929a010958f75fa.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4_Christmasdeals">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>371</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(139, 242, 115)</string>
   </property>
   <property name="text">
    <string>Christmas Deals 50-80% off for selected items</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4_pic4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>111</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Downloads/1774728327e8e8e4c375390ab856c85b.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3_pic3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>101</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Downloads/c7e74cc33b0dd999facd9c5daaae20db.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5_pic5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>121</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Downloads/2ec6cb3f0a6d09b4b63dfb47d8b234f8.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6pic6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>111</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Downloads/945d2d3715f0b727519aebca169cf585.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7pic7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>101</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Downloads/9fa0fda461b87091af1816a98f413438.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4_green">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>371</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Academy Engraved LET</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(139, 242, 115)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string> Make BMCC Green</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>520</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>8</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(139, 242, 115)</string>
   </property>
   <property name="text">
    <string>We commit that our Products are truly made from BMCC recylces to
 promote environmental sustainability within the fashion community at BMCC...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_aboutourmaterials">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>590</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baloo Tamma 2</family>
     <pointsize>13</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(139, 242, 115)</string>
   </property>
   <property name="text">
    <string>About Our Materials</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_logo_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Downloads/EcoWardrobe-01.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8pic8">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>590</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Downloads/fashion-design-class.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_description">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>620</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(139, 242, 115)</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt; color:#000000;&quot;&gt;â€œEco Wardrobe proudly incorporates recycled materials from the Borough of Manhattan Community College (BMCC) into our fashion line. By partnering with BMCC, we not only prioritize sustainability but also support local initiatives. These recycled materials undergo a meticulous process, transforming post-consumer waste into stylish, eco-friendly fabrics. This commitment not only reduces our environmental impact but also contributes to the circular economy. Choose Eco Wardrobe for fashion that tells a story of sustainability, community collaboration, and a brighter, greener future.&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>661</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2_Women">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Women</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7_contactus">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>760</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Contact Us - ecowardrobe.ew@gmail.com / 999-999-9990</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>790</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Privacy Policy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>790</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Return Policy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>790</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Shipping Policy</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Al Nile</family>
     <pointsize>13</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Al Nile'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:24pt; font-weight:700; color:#63d641;&quot;&gt;ECO WARDROBE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5_search">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Downloads/018428d1e3407a96da8665a62f85ca7e.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(161, 163, 163);</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButton_men</zorder>
  <zorder>pushButton_3_EcoCDeals</zorder>
  <zorder>label_2_pic1</zorder>
  <zorder>pushButton_4_Christmasdeals</zorder>
  <zorder>label_4_pic4</zorder>
  <zorder>label_3_pic3</zorder>
  <zorder>label_5_pic5</zorder>
  <zorder>label_6pic6</zorder>
  <zorder>label_7pic7</zorder>
  <zorder>label_4_green</zorder>
  <zorder>label_2</zorder>
  <zorder>label_aboutourmaterials</zorder>
  <zorder>label_8pic8</zorder>
  <zorder>textBrowser_description</zorder>
  <zorder>label_logo_2</zorder>
  <zorder>pushButton_2_Women</zorder>
  <zorder>label_7_contactus</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>textBrowser</zorder>
  <zorder>label_5_search</zorder>
  <zorder>lineEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
